chucks-pick3-web/angular-app/src/app/models/customer-info.model.ts

```diff
@@ -0,0 +1,35 @@
+import {CustomerInfo} from './customer-info';
+
+export class CustomerInfoModel implements CustomerInfo {
+  public firstName: string;
+  public lastName: string;
+  public email: string;
+  public phone: string;
+  public city: string;
+  public state: string;
+  public id: string;
+
+  constructor(config: CustomerInfo) {
+    if (config != null && config !== undefined) {
```

Comment:
Use ternary operator instead of if statement. 

[Ternary Operator](https://www.w3schools.com/js/js_comparisons.asp)

Reply:
Changed to ternary operator.